Core/DefaultVariantManager.swift

```diff
@@ -91,7 +91,7 @@ public protocol VariantRNG {
 public class DefaultVariantManager: VariantManager {
 
     public var currentVariant: Variant? {
-        let variantName = ProcessInfo.processInfo.environment["VARIANT", default: storage.variant ?? "" ]
```

Comment:
Out of curiosity, how come we changed from environment variable to launch argument?
Maestro lets set you the env variable as per [Maestro doc](https://maestro.mobile.dev/api-reference/configuration/flow-configuration)

Reply:
Ah, I just didn't know.  This is even better.  Will revert to the env version

Reply:
OK, had a look at this.

1/ Env doesn't appear to actually work.  I added some extra UI in the debug screen to check it (haven't committed that).

2/ We'd have to put it in every file since env is specific to the flow and I'm trying to have a standard 'set up' flow.

Interestingly the VARIANT argument gets changed to `-VARIANT` for some reason. Tested this with autoclear and that argument doesn't, but maybe because it's a bool?

Anyway, specifying an override prevents dax dialogs from showing and I think we want that because we want these tests to run against a something as close to production as possible.  So I am going to change the tests to just get past the onboarding and do everything else as expected.

Once we look to merge the new dax dialogs we'll have to update the tests again.

Reply:
Ah bummer, I saw that it was available so I suggested it. Thanks for the explanation, makes sense.